Documentation/api-reference.md

```diff
@@ -1,79 +1,100 @@
 # Analytics API Usage
 
+## Prerequisites
+
+Refer to the [Getting started guide](getting-started.md).
+
+## API reference
+
+- [clearQueue](#clearQueue)
+- [extensionVersion](#extensionVersion)
+- [EXTENSION](#EXTENSION)
+- [getQueueSize](#getQueueSize)
+- [getTrackingIdentifier](#getTrackingIdentifier)
+- [getVisitorIdentifier](#getVisitorIdentifier)
+- [sendQueuedHits](#sendQueuedHits)
+- [setVisitorIdentifier](#setVisitorIdentifier)
+------
+
+
```

Comment:
move L20 `This document details all the APIs provided by Analytics extension, along with sample code snippets on how to properly use the APIs.` to before the API list i.e on L9